faq.md

```diff
@@ -0,0 +1,99 @@
+ Sanitizer API - FAQs
+======================
+
+## How do I use the Sanitizer API
+
+Construct a new Sanitizer, using either the defaults or suppliying a custom
+configuration, and then pass in a string, a fragment, or an entire document
+for sanitization.
+
+```js
+const s = new Sanitizer();
+
+// Returns a DOM Fragment equivalent to "<em>Hello World</em>".
+s.sanitize("<em>Hello World</em><script>alert('surprise');</script>");
+```
+
+## Can I use the Sanitizer in my app?
+
+Not yet. Current browser support is preliminary and incomplete and still
+diverges in several details. And the specification itself is still in flux.
+We expect to be working through these issues in the coming months.
+
+If you wish to try it out already:
+
+* Firefox: Enable the `sanitizer.enabled` preference.
+* Chromium / Chrome: Start the browser with the
+  `--enable-blink-features=SanitizerAPI` flag.
```

Comment:
Can we put it behind chrome://flags/#enable-experimental-web-platform-features?

Reply:
Yes, we should. But that's a Chrome feature request, not a Sanitizer FAQ. :-)